readthedocsext/theme/templates/projects/partials/project_list.html

```diff
@@ -29,41 +29,20 @@
      class="ui primary button">{% trans "Learn how to get started" %}</a>
 {% endblock list_placeholder_text_empty %}
 
-{% block list_item_start %}
-  {# djlint:off #}
-  <tr data-bind="using: ProjectListItemView({id: {{ object.id }}, url: '{% url "projects-detail" object.slug %}'})">
-  {# djlint:on #}
-{% endblock list_item_start %}
-
 {% block list_item_right_menu %}
-  <div class="ui small icon buttons"
-       data-bind="event: {mouseover: fetch, focusin: fetch}">
-    {% blocktrans trimmed with project=object.name asvar label_link %}
-      View documentation for project {{ project }}
-    {% endblocktrans %}
-    <a class="ui {% if not object.has_good_build %}disabled{% endif %} button"
-       data-content="{{ label_link }}"
-       aria-label="{{ label_link }}"
-       data-bind="attr: {href: url_docs}"
-       target="_blank"
-       tabindex="{% if object.has_good_build %}0{% else %}-1{% endif %}">
-      <i class="fa-duotone fa-book icon"></i>
-    </a>
-
-    <button class="ui dropdown button">
-      <i class="fa-solid fa-ellipsis icon"></i>
-      <div class="menu">
-        <div class="header">{% trans "Admin" %}</div>
-        <a class="disabled item"
-           href="{% url "projects_edit" object.slug %}"
-           data-bind="css: { disabled: !is_admin() }">
-          <i class="fa-duotone fa-wrench icon"></i>
-          {% trans "Configure project" %}
-        </a>
-      </div>
-    </button>
-
-  </div>
+  <readthedocs-api csrf-token="{{ csrf_token }}"
+                   url="{% url "projects-detail" object.slug %}?expand=permissions">
+    <div class="ui small icon buttons">
+      <readthedocs-item-docs {% if not object.has_good_build %}disabled{% endif %}
+                             label="{% blocktrans with name=object.name %}View documentation for project {{ name }}{% endblocktrans %}">
+      </readthedocs-item-docs>
+      <button class="ui dropdown button">
+        <i class="fa-solid fa-ellipsis icon"></i>
+        <readthedocs-menu-project-admin class="menu" settings="{% url "projects_edit" object.slug %}">
```

Comment:
```suggestion
        <readthedocs-menu-project-admin class="menu" settings-url="{% url "projects_edit" object.slug %}">
```

I think `settings-url` is easier to read and follow. I was confused about what `settings` was when reading the JavaScript code, and I had to come back here to understand its value.